<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>firstDialog</class>
 <widget class="QDialog" name="firstDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>191</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Open New Window</string>
   </property>
  </widget>
  <widget class="QLabel" name="touchLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>205</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
